app/src/main/java/vn/tien/photo_world/data/model/Collection.java

```diff
@@ -0,0 +1,86 @@
+package vn.tien.photo_world.data.model;
+
+public class Collection {
+    private int mId;
+    private String mTitle;
+    private String mDescription;
+    private String mAuthor;
+    private int mTotalPhotos;
+    private Photo mCoverPhoto;
+    private CollectionLinks mLinks;
+
+    public Collection(int mId,
+                      String mTitle,
+                      String mDescription,
+                      String mAuthor,
+                      int mTotalPhotos,
+                      Photo mCoverPhoto,
+                      CollectionLinks mLinks) {
+        this.mId = mId;
+        this.mTitle = mTitle;
+        this.mDescription = mDescription;
+        this.mAuthor = mAuthor;
+        this.mTotalPhotos = mTotalPhotos;
+        this.mCoverPhoto = mCoverPhoto;
+        this.mLinks = mLinks;
+    }
+
+    public Collection() {
+    }
+
+    public int getmId() {
+        return mId;
+    }
+
+    public void setmId(int mId) {
```

Comment:
```suggestion
    public void setmId(int id) {
```
tương tự vs các phương thức khác